src/modules/i18n/index.js

```diff
@@ -18,6 +18,9 @@ export function langToLocal(lang) {
     return ['de', 'fr', 'it', 'rm'].includes(lang) ? `${lang}-CH` : lang
 }
 
+export function localToLang(local) {
+    return local.replace('-CH', '')
+
```

Comment:
I would even go further, and change things for all local (`en-US` or `en-UK` is not covered by our use case, neither is `de-DE` or `fr-FR`)

Reply:
Done